features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_MCU_NRF51822/source/nRF5xGap.cpp

```diff
@@ -274,6 +274,8 @@ ble_error_t nRF5xGap::stopAdvertising(void)
     return BLE_ERROR_NONE;
 }
 
+/* Central role is not supported by S112, return BLE_ERROR_NOT_IMPLEMENTED */
+#ifndef S112
```

Comment:
Does this #ifdef still matter if NRF_SDH_BLE_CENTRAL_LINK_COUNT is being set to 0?

Reply:
I think so, in general the `#ifndef S112`'s were added wherever the stack tried to call functions which nordic simply don't define for the 810